src/main/scala/swiss/dasch/infrastructure/CommandExecutorLive.scala

```diff
@@ -53,21 +53,29 @@ trait CommandExecutor {
 }
 
 object CommandExecutor {
-  def buildCommand(command: String, params: String): ZIO[CommandExecutor, Nothing, Command] =
-    ZIO.serviceWithZIO[CommandExecutor](_.buildCommand(command, params))
+  def buildCommand(command: String, params: String*): ZIO[CommandExecutor, Nothing, Command] =
+    ZIO.serviceWithZIO[CommandExecutor](_.buildCommand(command, params: _*))
 }
 
 final case class CommandExecutorLive(sipiConfig: SipiConfig, storageService: StorageService) extends CommandExecutor {
 
-  override def buildCommand(command: String, params: String): UIO[Command] =
+  override def buildCommand(command: String, params: String*): UIO[Command] =
     if (sipiConfig.useLocalDev) {
       for {
         assetDir <- storageService.getAssetsBaseFolder().flatMap(_.toAbsolutePath).orDie
-      } yield Command(
-        s"docker run --entrypoint $command -v $assetDir:$assetDir daschswiss/knora-sipi:${BuildInfo.sipiVersion} $params"
-      )
+      } yield {
+        Command(
+          List(
+            List("docker", "run", "--entrypoint", command),
+            List("-v", s"$assetDir:$assetDir"),
+            List(s"daschswiss/knora-sipi:${BuildInfo.sipiVersion}"),
```

Comment:
Could these two interpolations also be replaced or it doesn't make sense? I'm not sure are these variables strings already or not, that's why added `toString` conversion to them - remove if not necessary.
```suggestion
            List("-v", "assetDir.toString, ":", "assetDir.toString"),
            List("daschswiss/knora-sipi:", "BuildInfo.sipiVersion.toString"),
```

Reply:
That would change the meaning – they have to be in the same argument, now they would be in two.

A very imperfect analogy would be `ls directory/file` vs `ls directory file`. The analogy here is that the space would split the arguments into two, which would change the meaning of code entirely.

Reply:
As far as it does not expose it to vulnerabilities mentioned in the task I'm fine with what you did :)